fpsim/empowerment.py

```diff
@@ -0,0 +1,289 @@
+"""
+Methods and functions related to empowerment and education
+TODO: maybe move education stuff to another module, education.py
+"""
+
+# %% Imports
+import numpy as np  # Needed for a few things not provided by pl
+import sciris as sc
+from . import utils as fpu
+from . import defaults as fpd
+
+
+# %% Initialization methods
+
+
+def init_urban_states(ppl):
```

Comment:
Renamed by appending `_states` in the function name because these methods will initialise People's attribute equivalent to starsim's states, using values other than those specified in defaults.py. Any logic about using different strategies to initialise the specific state could go here.

```python
init_empowerment_states(ppl):
    if ppl.pars['education'] is not None:
         get_edu_vals_from_data(...)
    else:
         get_edu_vals_from_other_source(...) -- some other distribution
          
   ```